src/windows/views/timeline_webview.py

```diff
@@ -2537,13 +2537,19 @@ def PlayheadMoved(self, position_frames):
             self.window.previewFrame(position_frames)
 
     @pyqtSlot(int)
-    def movePlayhead(self, position_frames):
+    def movePlayhead(self, position_frames, followPlayhead = False):
```

Comment:
This should be `@pyqtSlot(int, bool)` to match the function.

Actually, given that `followPlayhead` is optional, it'd be appropriate to decorate it both ways [("chain" the decorators)](https://www.riverbankcomputing.com/static/Docs/PyQt5/signals_slots.html#the-pyqtslot-decorator):
```python
@pyqtSlot(int)
@pyqtSlot(int, bool)
def movePlayhead(...):
```

(Though, here again, I'm not sure the decoration actually has any purpose.)

Reply:
Thanks for the tip about chaining decorators.  Although this suggestion is overcome by moving the followPlayhead functionality to controller.js, I appreciate the education.